app/scripts/templates/settings/clients.mustache

```diff
@@ -26,7 +26,8 @@
             {{^isCurrentDevice}}
             <div class="last-connected">{{lastAccessTimeFormatted}}</div>
             {{/isCurrentDevice}}
-            <button class="settings-button warning client-disconnect" data-id="{{id}}" data-type="{{clientType}}">{{#t}}Disconnect…{{/t}}</button>
+            <button class="settings-button warning client-disconnect" data-id="{{id}}" data-type="{{clientType}}">
+            {{#isDevice}}{{#t}}Disconnect…{{/t}}{{/isDevice}}{{^isDevice}}{{#t}}Disconnect{{/t}}{{/isDevice}}</button>
```

Comment:
@divyabiyani you can indent the two `{{#t}}Disconnect` by 2 spaces to the right. Thanks for making the updates!